packages/design-system-tokens/src/themes/core-component.ts

```diff
@@ -298,9 +298,10 @@ export const components: AnyTokenValues = {
 
   'dropdown': {
     '__background-color':                         t.color['white'],
-    '__icon-size':                                '10px',
+    '__icon-size':                                '24px',
     '-item__border-width':                        '1px',
     '-item__border-color':                        t.color['gray-lighter'],
+    '-menu__border-radius--open':                 '3px',
```

Comment:
Since this should always match the border radius of the ds-c-field button, I don't actually think we should track this as a separate variable.